Objects/object.c

```diff
@@ -2107,6 +2107,12 @@ _PyTrash_thread_destroy_chain(void)
          * up distorting allocation statistics.
          */
         assert(op->ob_refcnt == 0);
+        /* The deallocator executing this function still occupies
```

Comment:
I am confused about the first sentence, "The deallocator executing this function still occupies the stack". Otherwise, LGTM.